tests/whois_ip_agent_test.py

```diff
@@ -75,3 +75,31 @@ def testAgentWhoisIP_whenDnsAAAAMsgRecieved_emitsWhoisRecords(scan_message_dns_a
 
     assert len(agent_mock) == len(scan_message_dns_aaaa_record.data['values'])
     assert agent_mock[0].selector == 'v3.asset.ip.v6.whois'
+
+
+def testAgentWhoisIP_whenIPv4WithMaskTarget_returnsWhoisRecord(
```

Comment:
Please don't forget to add a check for scanning /24 and /23 for instance scanned before, so we don't scan again.